Firestore/Port/string_util_test.cc

```diff
@@ -27,7 +28,7 @@ TEST(Util, PrefixSuccessor) {
   EXPECT_EQ(PrefixSuccessor("a"), "b");
   EXPECT_EQ(PrefixSuccessor("aaAA"), "aaAB");
   EXPECT_EQ(PrefixSuccessor("aaa\xff"), "aab");
-  EXPECT_EQ(PrefixSuccessor(string("\x00", 1)), "\x01");
+  EXPECT_EQ(PrefixSuccessor(std::string("\x00", 1)), "\x01");
   EXPECT_EQ(PrefixSuccessor("az\xe0"), "az\xe1");
   EXPECT_EQ(PrefixSuccessor("\xff\xff\xff"), "");
```

Comment:
Changed to NE and got the red X in the right place with the right failure message. 👍